docs/proposals/container-runtime-interface/capabilities.md

```diff
@@ -0,0 +1,137 @@
+
+```shell
+$ kubectl get pod -o yaml
+apiVersion: v1
+items:
+- apiVersion: v1
+  kind: Pod
+  metadata:
+  ...
+  spec:
+    containers:
+    - command:
+      - sleep
+      - "1000"
+      image: busybox
+      imagePullPolicy: Always
+      name: my-busybox
+      securityContext:
+        capabilities:
+          default:
+          - SETPCAP, MKNOD, AUDIT_WRITE, ...
+          add:
+          - NET_ADMIN
+          drop:
+          - MKNOD
+      ...
+  status:
+    containerStatuses:
+    - image: busybox
+      name: my-busybox
+      state:
+        running:
+          startedAt: 2016-09-27T22:33:38Z
+    ...
+```
+
+The `PodSecurityPolicySpec` gets a new `DefaultCapabilities` field. The existing fields `DefaultAddCapabilities`, and `RequiredDropCapabilities` will be marked as deprecated.
+
+```
+type PodSecurityPolicySpec struct {
+...
+        // DefaultAddCapabilities is the default set of capabilities that will be added to the container
+        // unless the pod spec specifically drops the capability. You may not list a capability in both
+        // DefaultAddCapabilities and RequiredDropCapabilities.
+        // Note: this field is deprecated in favor of the DefaultCapabilities field.
+        DefaultAddCapabilities []api.Capability `json:"defaultAddCapabilities,omitempty"`
+        // RequiredDropCapabilities are the capabilities that will be dropped from the container.  These
+        // are required to be dropped and cannot be added.
+        // Note: this field is deprecated in favor of the DefaultCapabilities field.
+        RequiredDropCapabilities []api.Capability `json:"requiredDropCapabilities,omitempty"`
+        // AllowedCapabilities is a list of capabilities that can be requested to add to the container.
+        // Capabilities in this field may be added at the pod author's discretion.
+        // You must not list a capability in both AllowedCapabilities and RequiredDropCapabilities.
+        AllowedCapabilities []api.Capability `json:"allowedCapabilities,omitempty"`
+        // DefaultCapabilities is a list of capabilities that will always be applied to the container.
+        // In addition to the capabilities listed here, the pod author can add additional capabilities
+        // specified in the AllowedCapabilities field.
+        DefaultCapabilities []api.Capability `json:"defaultCapabilities,omitempty"`
+...
+}
+```
+
+The `Capabilities` struct in `pkg/api/types.go`, and `pkg/api/v1/types.go` gets a new `Default` field:
+
+```
+// Capabilities represent POSIX capabilities that can be added or removed to a running container to/from a default set.
+type Capabilities struct {
+        // Added capabilities to the default capabilities
+        Add []Capability `json:"add,omitempty"`
+        // Removed capabilities from the default capabilities
+        Drop []Capability `json:"drop,omitempty"`
+        // Default capabilities
+        Default []Capability `json:"default,omitempty"`
```

Comment:
As mentioned above, I'd prefer for this to be the resolved whitelist. From the user perspective it makes it easier to reason about.